### PR TITLE
Removes additional columns in test_owners.csv

### PR DESCRIPTION
**Pull request by calebamiles**
**What this PR does / why we need it**:

fixes a huge collection of typos with the number of columns in the CSV file that probably has broken the auto assign bot

**Special notes for your reviewer**:

None

**Release note**:

`NONE`

### CODE REVIEW

**Comment by calebamiles** [comment]
Sorry for breaking the bot!

**cc: @ethernetdan, @pwittrock, @kubernetes/test-infra-reviewers **

**Comment by liggitt** [comment]
looks like it also adds about 200 lines... that intentional?

File: test/test_owners.csv [hunk 1]
```diff
@@ -1,535 +1,738 @@
...
+Garbage collector should delete pods created by rc when not orphaning,justinsb,1,cluster-lifecycle
+Garbage collector should orphan RS created by deployment when deleteOptions.OrphanDependents is true,rkouj,0,cluster-lifecycle
+Garbage collector should orphan pods created by rc if delete options say so,fabioy,1,cluster-lifecycle
+Garbage collector should orphan pods created by rc if deleteOptions.OrphanDependents is nil,zmerlynn,1,cluster-lifecycle
+"Generated release_1_5 clientset should create pods, delete pods, watch pods",rrati,0,api-machinery
+"Generated release_1_5 clientset should create v2alpha1 cronJobs, delete cronJobs, watch cronJobs",soltysh,1,api-machinery
+HA-master survive addition/removal replicas different zones,derekwaynecarr,0,api-machinery
+HA-master survive addition/removal replicas multizone workers,rkouj,0,api-machinery
+HA-master survive addition/removal replicas same zone,derekwaynecarr,0,api-machinery
+AppArmor should enforce an AppArmor profile,derekwaynecarr,0,node
```
Comment (ethernetdan) [inline test/test_owners.csv]: @calebamiles Looks like the file got repeated here. Could it be an issue with the [`hack/update_owners.py`](https://github.com/kubernetes/kubernetes/blob/master/hack/update_owners.py) script?

**Comment by calebamiles** [comment]
- @ethernetdan, should be fixed
- @liggitt, nope wasn't intentional; should be fixed now

**Comment by ethernetdan** [comment]
@k8s-bot gci gke e2e test this
@k8s-bot cvm gke e2e test this
@k8s-bot cvm gce e2e test this

**Comment by ethernetdan** [comment]
@k8s-bot non-cri e2e test this
@k8s-bot gce etcd3 e2e test this
@k8s-bot gci gce e2e test this

**Comment by ethernetdan** [comment]
Related to kubernetes/test-infra#1935

**Comment by ethernetdan** [comment]
@rmmh @liggitt does this look good now?

**Pull request merged**